resources/views/home.blade.php

```diff
@@ -1,23 +1,20 @@
 @extends('layouts.app')
-
 @section('content')
 <div class="container">
-    <div class="row justify-content-center">
-        <div class="col-md-8">
-            <div class="card">
-                <div class="card-header">{{ __('Dashboard') }}</div>
-
-                <div class="card-body">
-                    @if (session('status'))
-                        <div class="alert alert-success" role="alert">
-                            {{ session('status') }}
-                        </div>
-                    @endif
-
-                    {{ __('You are logged in!') }}
-                </div>
-            </div>
-        </div>
+ <div class="row justify-content-center">
+  <div class="col-md-8">
+   <div class="card">
+    <div class="card-header">{{ __('Dashboard') }}</div>
+    <div class="card-body">
+     @if (session('status'))
+     <div class="alert alert-success" role="alert">
+      {{ session('status') }}
+     </div>
+     @endif
+     {{ __('You are logged in!') }}
```

Comment:
{{ __('Dashboard') }}
 {{ __('You are logged in!') }}
 lấy từ đâu nhỉ?

Reply:
i18n đó a